types/queries.go

```diff
@@ -111,6 +113,64 @@ type AllBalancesResponse struct {
 	Amount Coins `json:"amount"`
 }
 
+type IbcQuery struct {
+	PortId       *PortIdQuery       `json:"port_id,omitempty"`
```

Comment:
Req: `ID` should be upper case . This and other places

Reply:
Translating from Rust. Missed that one.
Asking again. For Go, do you prefer `IBCQuery` or `IbcQuery`. I can do either and make them consistent.